apps/src/sites/studio/pages/home/_homepage.js

```diff
@@ -165,3 +166,39 @@ function showHomepage() {
     document.getElementById('homepage-container')
   );
 }
+
+window.CleverTakeoverManager = function (options) {
+  this.options = options;
+  const self = this;
+
+  const linkCleverDiv = $('<div>');
+  function showLinkCleverModal(cancel, submit) {
+    $(document.body).append(linkCleverDiv);
+
+    ReactDOM.render(
+      <LinkCleverAccountModal
+        isOpen={true}
+        handleCancel={cancel}
+        handleSubmit={submit}
+      />,
+      linkCleverDiv[0]
+    );
+  }
+
+  if (self.options.cleverLinkFlag) {
+    showLinkCleverModal(onCancelModal, onConfirmLink);
+  }
+
+  function closeLinkCleverModal() {
+    ReactDOM.unmountComponentAtNode(linkCleverDiv[0]);
+  }
+
+  function onCancelModal() {
+    $("#user_user_type").val("student");
+    closeLinkCleverModal();
+  }
+
+  function onConfirmLink() {
+    window.location.href = "/users/clever_takeover?mergeID=" + self.options.userIDToMerge + "&token=" + self.options.mergeAuthToken;
```

Comment:
Is the `mergeAuthToken` safe to send to the client?

Reply:
I'm going to do a security review on this before pushing live, because there are some other things I worry a lot about with this flow.